### PR TITLE
[EA] Rollout people directory

### PR DESCRIPTION
**Pull request by oetherington**
Rollout the people directory for all users. [Relevant Figma](https://www.figma.com/design/qAHRGADVOGojqKpqbGMFFu/2024-Q2?node-id=939-11312&m=dev).

In this PR:

- Release people directory for all users
- Update BETA menu tag to NEW as per figma
- Update "beta" copy on people directory page to "new"

Constituent PRs:

- [x] Map view (#9342)
- [x] Checkbox redesign (#9367)
- [x] Fix overlapping icons bug (#9366)
- [x] Show topic interests on user profile (#9368)
- [x] Edit profile page redesign (#9373)
- [x] Topic interests column and filtering (#9421)
- [x] Comments column (#9425)
- [x] Posts column (#9429)
- [x] Sorting dropdown (#9444)
- [x] Mobile UI (#9473)
- [ ] Show search highlights (not needed for initial release)
- [ ] Select all matching filters (not needed for initial release)

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1207445502383053) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by oetherington** [comment]
FYI @agnesstenlund

File: packages/lesswrong/lib/cookies/cookies.ts [hunk 1]
```diff
@@ -176,6 +176,14 @@ export const HIDE_EAG_BANNER_COOKIE = registerCookie({
   description: "Don't show any EAG(x) banners",
 });
 
+export const NAV_MANU_FLAG_COOKIE_PREFIX = "nav_menu_flag_";
```
Comment (Will-Howard) [inline packages/lesswrong/lib/cookies/cookies.ts:179]:
```suggestion
export const NAV_MENU_FLAG_COOKIE_PREFIX = "nav_menu_flag_";
```

File: packages/lesswrong/components/common/TabNavigationMenu/TabNavigationItem.tsx [hunk 2]
```diff
@@ -118,24 +120,82 @@ export type TabNavigationItemProps = {
...
+  const cookieName = `${NAV_MANU_FLAG_COOKIE_PREFIX}${tab.id}_${tab.flag}`;
+  const [cookies, setCookie] = useCookiesWithConsent();
+  const cookie = cookies[cookieName];
+  const flag = tab.flag;
+  if (flag === "new") {
+    const {clickedAt, firstViewedAt} = parseCookie(
+      cookie,
+      (value: string) => setCookie(cookieName, value),
+    );
+    const MS_PER_MONTH = 2_628_000_000;
```
Comment (Will-Howard) [inline packages/lesswrong/components/common/TabNavigationMenu/TabNavigationItem.tsx:162]: Could you make this 28 days? A month is ambiguous

**Review by Will-Howard — APPROVED** [review]
LGTM

**Pull request merged**